gpustack/scheduler/calculator.py

```diff
@@ -164,113 +127,282 @@ def _gguf_parser_env(model: Model) -> dict:
+            action='store_false',
+            dest="image_autoencoder_tiling",
+            required=False,
+        )
+        parser.add_argument(
+            "--image-batch-count",
+            "--batch-count",
+            "--image-max-batch",
+            type=int,
+            required=False,
+        )
+        parser.add_argument(
+            "--image-free-compute-memory-immediately",
+            action='store_true',
+            required=False,
+        )
+        parser.add_argument(
+            "--image-height",
+            "--height",
+            "--image-max-height",
+            type=int,
+            required=False,
+        )
+        parser.add_argument(
+            "--image-no-autoencoder-offload",
+            "--vae-on-cpu",
+            "--image-no-vae-model-offload",
+            action='store_true',
+            required=False,
+        )
+        parser.add_argument(
+            "--image-no-conditioner-offload",
+            "--clip-on-cpu",
+            "--image-no-text-encoder-model-offload",
+            action='store_true',
+            required=False,
+        )
+        parser.add_argument(
+            "--image-no-control-net-offload",
+            "--control-net-cpu",
+            "--image-no-control-net-model-offload",
+            action='store_true',
+            required=False,
+        )
+        parser.add_argument(
+            "--image-width",
+            "--width",
+            "--image-max-width",
+            type=int,
+            required=False,
+        )
+        # Load
+        parser.add_argument(
+            "--cache-expiration",
+            type=str,
+            required=False,
+        )
+        parser.add_argument(
+            "--skip-cache",
+            action='store_true',
+            required=False,
+        )
+        parser.add_argument(
+            "--skip-dns-cache",
+            action='store_true',
+            required=False,
+        )
+        parser.add_argument(
+            "--skip-proxy",
+            action='store_true',
+            required=False,
+        )
+        parser.add_argument(
+            "--skip-range-download-detect",
+            action='store_true',
+            required=False,
+        )
+        parser.add_argument(
+            "--skip-tls-verify",
+            action='store_true',
+            required=False,
+        )
+
+        try:
+            args_parsed = parser.parse_known_args(args=args)
+            for attr_name in [attr.name for attr in dataclasses.fields(self.__class__)]:
+                if attr_value := getattr(args_parsed[0], attr_name):
+                    try:
+                        setattr(self, attr_name, attr_value)
+                    except ValueError:
+                        # If the attribute is not a valid type, skip it
+                        pass
+        except (argparse.ArgumentError, argparse.ArgumentTypeError):
+            # If the argument parsing fails, skip it
+            pass
```

Comment:
What use cases lead to these exceptions, should we log anything useful for debugging?